pkg/apis/core/v1alpha1/federatedtypeconfig_types_test.go

```diff
@@ -51,7 +51,7 @@ var _ = Describe("FederatedTypeConfig", func() {
 	Describe("when sending a storage request", func() {
 		Context("for a valid config", func() {
 			It("should provide CRUD access to the object", func() {
-				client = cs.CoreV1alpha1().FederatedTypeConfigs()
+				client = cs.CoreV1alpha1().FederatedTypeConfigs("foo")
```

Comment:
Just a question here, why did the `kubebuilder` did not update the test file automatically when change from cluster scope to namespace scope?

Reply:
kubebuilder doesn't maintain generated files.

Reply:
I see, thanks @marun

Reply:
These files are generated as scaffolding when you first create a resource type, but no longer maintained when regenerating code using kubebuilder. Perhaps that could be a feature in kubebuilder, but often times you wouldn't want to regenerate these unit tests if you've heavily modified the unit tests after the fact.